model/task/task.go

```diff
@@ -1949,6 +1949,8 @@ func AbortVersion(versionId string, reason AbortInfo) error {
 	}
 	if reason.TaskID != "" {
 		q[IdKey] = bson.M{"$ne": reason.TaskID}
+		// if the aborting task is part of a display task, we also don't want to mark it as aborted
+		q[ExecutionTasksKey] = bson.M{"$ne": reason.TaskID}
```

Comment:
Apologies if this is a stupid question, but why is this on ExecutionTasks and not on DisplayTask? I thought we want to avoid marking the display task as aborted if one of its execution tasks has a legitimate failure.

Reply:
The example in the test might also help! The display task has three execution tasks; the second one et2 fails, but we don't mark the display task as aborted because it has ExecutionTasks = ["et1", "et2", "et3"]